python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
+                      'something that isn\'t a str, list, or tuple.',
+             'W0101': 'Code after a return or raise statement will '
+                      'never be run, so either the function is '
+                      'returning early or you should remove it.',
+             'W0102': 'Dangerous default value %s as argument. '
+                      'Using mutable values as a default argument '
+                      'is a common cause of confusing and unwanted behaviour.',
+             'W0104': 'This statement doesn\'t have any effect, and could be '
+                      'removed without changing the behaviour of the program.',
+             'W0106': 'Expression "%s" isn\'t assigned to a variable.',
+             'W0108': 'The lambda function isn\'t necessary since the lambda '
+                      'arguments are only being passed into a function call.',
+             'W0109': 'Duplicate key %r in dictionary. '
+                      'When constructing a dictionary, the last value assigned '
+                      'to a key takes precedence, making previous ones redundant.',
+             'W0122': 'Using `exec()` to evaluate a code-block represented as a '
+                      'str can be potentially dangerous, and is typically a sign of poor design.',
+             'W0123': 'Using `eval()` to evaluate an expression represented as a '
+                      'str can be potentially dangerous, and is typically a sign of poor design.',
+             'W0199': 'Calling assert on a tuple will only evaluate to false if '
+                      'it is empty. It\'s likely you intended \'assert x, y\' rather '
+                      'than \'assert (x,y)\'.'
+             },
+        'ComparisonChecker':
+            {'R0123': 'tHE \'is\' keyword compares two objects location in memory rather '
+                      'than value equality. When performing comparison with a literal, '
+                      'this is usually not what you want. Did you mean \'==\'?',
+             'C0123': 'You should use \'isinstance\' rather than \'type\' to '
+                      'perform a typecheck.'
+             },
+        'NameChecker':
+            {'C0102': 'Black listed name "%s". You should give your variables '
+                      'meaningful rather than generic names.',
+             'C0103': '%s name "%s" doesn\'t conform to %s. '
+                      'Please refer to the PyTA Help Documentation for more information '
+                      'on Python Naming Conventions.'
+             },
         'PassChecker':
-             {'W0107': 'Unnecessary pass statement (you should remove this)'}
+            {'W0107': 'Unnecessary pass statement (you should remove this). '
+                      'A pass statement should only be used to represent '
+                      'an \"empty\" code block, where nothing happens.'},
+        'BasicErrorChecker':
+            {'E0101': '__init__ is called by the special method __new__ '
```

Comment:
This is too technical. Replace with "__init__ is called only to initialize the attributes of a new object. It should not return anything."